code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -204,6 +204,7 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 		user.put_in_hands(inserted_pai)
 		balloon_alert(user, "removed pAI")
 		inserted_pai = null
+		update_overlays()
```

Comment:
You're meant to call `update_appearance(UPDATE_ICON)`, `update_overlays` is a pure proc

Reply:
whats a pure proc?

Reply:
It shouldn't have side effects on its own, it just returns the new overlays